tests/topotests/bgp_as_wide_bgp_identifier/test_bgp_as_wide_bgp_identifier.py

```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/env python
+
+#
+# test_bgp_as_wide_bgp_identifier.py
+# Part of NetDEF Topology Tests
+#
+# Copyright (c) 2020 by
+# Donatas Abraitis <donatas.abraitis@gmail.com>
+#
+# Permission to use, copy, modify, and/or distribute this software
+# for any purpose with or without fee is hereby granted, provided
+# that the above copyright notice and this permission notice appear
+# in all copies.
+#
+# THE SOFTWARE IS PROVIDED "AS IS" AND NETDEF DISCLAIMS ALL WARRANTIES
```

Comment:
mm, are you netdef? :)

Reply:
afaik, no 🦆

Reply:
should I replace this in all the tests I did? :)

Reply:
@qlyoung :point_up: